src/debugAdapter/runtimeInterface.ts

```diff
@@ -295,4 +280,32 @@ export default class RuntimeInterface extends EventEmitter {
       throw new Error('Debug session is undefined');
     }
   }
+
+  /**
+   * Validate if the network parameter was set. Case the network is empty,
+   * a temp network is created to generate a provider url
+   *
+   * @param config Represents the truffle configuration file.
+   * @param args Represents the arguments needed to initiate a new Truffle `DebugSession` request.
+   * @returns
+   */
+  private validateNetwork(config: Config, args: Required<DebuggerTypes.DebugArgs>): void {
```

Comment:
This method is not using any property on `this`. Should we make it `static`? Or we even could move it outside this class.